modelina-website/src/pages/api/functions/CSharpGenerator.ts

```diff
@@ -32,7 +32,15 @@ export async function getCSharpModels(
   if (generatorOptions.csharpIncludeJson) {
     options.presets.push(CSHARP_JSON_SERIALIZER_PRESET)
   }
-  if (generatorOptions.csharpIncludeNewtonsoft) {
+  if(generatorOptions.csharpOverwriteEqual){
```

Comment:
You need to add this as part of https://github.com/asyncapi/modelina/blob/6ff385a85c7087d34d780fe224c7c53ccdb3d1a6/modelina-website/src/pages/api/functions/CSharpGenerator.ts#L35

Or set the following options in this statement:

```
{
      preset: CSHARP_COMMON_PRESET,
      options: {
        equal: generatorOptions.csharpOverwriteEqual,
        hashCode: false
      }
    }
```

Reply:
But in the documentation of [Csharp.md](https://github.com/asyncapi/modelina/blob/master/docs/languages/Csharp.md#generate-models-with-equals-and-gethashcode-methods), its given as use the preset `CSHARP_COMMON_PRESET` and provide the options `equal: true` and `hashCode: true`. Could you once check and let me know?

Reply:
Thats only when you want to include both the overwriting of equals method as well as hashcode, otherwise you can conditionally control it by setting one of them to false.

Reply:
> Thats only when you want to include both the overwriting of equals method as well as hashcode, otherwise you can conditionally control it by setting one of them to false.

Got it @jonaslagoni! Thank you for clearing it. 🙂